utils/delete_workflows.sh

```diff
@@ -0,0 +1,7 @@
+export OWNER="CraftsmenLtd"
```

Comment:
You don't need to export these variables as you'll use it immediately after. Can get away with an assignment